package/CHANGELOG

```diff
@@ -131,7 +131,10 @@ Enhancements
     'protein' selection (#2751 PR #2755)
   * Added an RDKit converter that works for any input with all hydrogens
     explicit in the topology (Issue #2468, PR #2775)
-  
+  * Added warning message and test for when user provided
+    grid dimensions are ignored. (Issue #3148, PR #3154)
```

Comment:
Can be updated to reflect it'll now raise an error. I'd also clarify the root cause as being when grid center wasn't defined.